src/main/java/core/basesyntax/DrawArea.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface DrawArea extends GetArea {

```

Comment:
In task explanation:
To 'draw' means to print out all information about a figure using System.out.println() (you shouldn't override toString() method for this).
You don't need to extend from another interface, and don't need to create dafault logic, you need to override this method in all figures class